README.md

```diff
@@ -187,6 +187,21 @@ export apiaudio_key=<your-key>
 
 If you provide both environment variable and `apiaudio.api_key` authentication, the `apiaudio.api_key` will be used.
 
+### Super Organizations
+
+In order to control a child organization of yours, please use the following method to assume that organization id.
+
+Set your child organization id to `None` to stop assuming an organization.
+
+```python
+import apiaudio
+
+apiaudio.set_assume_org_id('child_org_id')
+
+# Stop using
+apiaudio.set_assume_org_id(None)
```

Comment:
Why is None not a string?

Reply:
And why None - and not false?

Reply:
simply thinking, i assume no org, so i am myself. or i assume an org.

Reply:
For syntax and usability we could have a reset_assume_org_id() ? 

its an extra function but it makes it very clear,

whereas set_assume_org_id(false) or set_assume_org_id("") actually implies you don't use a org at all